keyboards/keycapsss/kimiko/config.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2019 MechMerlin
+ * Copyright 2023 @Ex3c4Def
+ * Copyright 2023 @BenRoe (keycapsss.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// OLED definitions
+#ifdef OLED_ENABLE
+#   define OLED_FONT_H "keyboards/keycapsss/kimiko/glcdfont.c"
+#endif
```

Comment:
this is conflicting with the rev1 default keymap. 

This should be moved to post_config.h, and:
```suggestion
#ifndef OLED_FONT_H
#   define OLED_FONT_H "keyboards/keycapsss/kimiko/glcdfont.c"
#endif
```